app/parse.py

```diff
@@ -15,8 +40,76 @@ class Product:
     num_of_reviews: int
 
 
+PRODUCT_FIELDS = [field.name for field in fields(Product)]
+
+
+def get_webdriver() -> webdriver.Chrome:
+    chrome_options = Options()
+    chrome_options.add_argument("--headless")
+    chrome_options.add_argument("--disable-gpu")
+    chrome_options.add_argument("--no-sandbox")
+    driver = webdriver.Chrome(options=chrome_options)
```

Comment:
Ensure that the ChromeDriver is correctly installed and configured on the system where this script will run. Without it, the `webdriver.Chrome()` call will fail.